src/matlab/reactormethods.cpp

```diff
@@ -48,6 +48,9 @@ void reactormethods(int nlhs, mxArray* plhs[],
         case 7:
             iok = reactor_setKineticsMgr(i, int(v));
             break;
+        case 8:
+            iok = reactor_setChemistry(i, bool(v));
```

Comment:
This casting of `double` to `bool` is recommended by the MathWorks documentation for the `mxGetScalar` function (which the current `getDouble` is a wrapper around). See: http://www.mathworks.com/help/matlab/apiref/mxgetscalar.html